plugins/PostProcessingPlugin/scripts/Stretch.py

```diff
@@ -452,6 +452,8 @@ class Stretch(Script):
     def __init__(self):
         super().__init__()
 
+        self._application = Application.getInstance()
```

Comment:
Why do we need to cache this? `Application.getInstance()` is just as nice, and there will only be a single source of truth.